wgs/wgs-gather-vcfs/wgs-gather-vcfs.inputs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "##_COMMENT1": "Path to the intervals file",
+  "GatherVcfsCloud.intervals_file": "__INSERT_GCS_PATH_TO_INTERVALS_FILE__",
+
+  "##_COMMENT2": "Set which chromosomes to gather, which shards to ignore, and an output suffix value.",
+  "GatherVcfsCloud.chromosomes": ["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","X","Y"],
+  "GatherVcfsCloud.shards_to_ignore": [],
+  "GatherVcfsCloud.output_suffix": "_combined_from_cloud",
```

Comment:
What was the motivating use case for the output_suffix?

Reply:
It was useful while testing the gatk GatherVcfsCloud at first to differentiate between test runs. Looking at this now I think its safe to remove.